docs/src/prototypes/CopyToClipboard/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import { Flex, Text, Provider, themes } from '@stardust-ui/react/src'
+import CopyToClipboard from './CopyToClipboard'
+import { PrototypeSection, ComponentPrototype } from '../Prototypes'
+
+class CopyToClipboardPrototypeAttached extends React.Component {
+  render() {
+    const commitID = '3422f7d'
+    return (
+      <Provider theme={themes.teams} style={{ padding: 10 }}>
```

Comment:
I've seen this multiple times, can you please clarify why are you using `Provider`?
You don't need it there as doc site uses `teams` theme by default.

To specify `padding`:
- you can use cool old `div`s: `<div style={{ padding: 10 }} />`
- you can specify it on `Flex`: `<Flex styles={{ padding: '10px' }} />`
- you can use `padding` prop on `Flex`: `<Flex padding="padding.medium" />`, this is the best probably

Reply:
When I thought of an element that would wrap the content, I thought of Provider. It's wrong though, thank you for letting me know.

Reply:
`Provider` does more more than just content wrap, it merges top-level and its context that include themes and other things 🔬 

_Usually_, you don't more than one `Provider` per app.

Reply:
I also thought that `<>` is a short for `Provider`. So I decided to choose `Provider` to make it more clear for me